### PR TITLE
Update and fix Equipment List related

### PR DESCRIPTION
**Pull request by sinsinpub**
Fixes:
 * Scraped or consumed gear not updated to local storage in time

Updates:
 * Add support to check item from ship's ex-slot
 * Add type 38 for new fighter

### CODE REVIEW

File: src/library/modules/Kcsapi.js [hunk 1]
```diff
@@ -1377,6 +1377,7 @@ Previously known as "Reactor"
 					rsc[i] += x;
 				});
 				KC3GearManager.remove( itemId );
+				KC3GearManager.save();
```
Comment (dragonjet) [inline src/library/modules/Kcsapi.js]: we could probably move the `save()` outside of the loop

Comment (sinsinpub) [inline src/library/modules/Kcsapi.js]: right, amended

**Comment by dragonjet** [comment]
:+1:

**Comment by sinsinpub** [comment]
btw, the item on ex slot issue has been reported at #1140

**Pull request merged**